pkg/monitoring/daemonsets.go

```diff
@@ -5,6 +5,7 @@ package monitoring
 
```

Comment:
copyright 2021

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2